pyomo/repn/tests/ampl/test_nlv2.py

```diff
@@ -0,0 +1,313 @@
+#  ___________________________________________________________________________
+#
+#  Pyomo: Python Optimization Modeling Objects
+#  Copyright (c) 2008-2022
+#  National Technology and Engineering Solutions of Sandia, LLC
+#  Under the terms of Contract DE-NA0003525 with National Technology and
+#  Engineering Solutions of Sandia, LLC, the U.S. Government retains certain
+#  rights in this software.
+#  This software is distributed under the 3-clause BSD License.
+#  ___________________________________________________________________________
+#
+
+import pyomo.common.unittest as unittest
+
+import math
+
+import pyomo.repn.plugins.nl_writer as nl_writer
+from pyomo.common.log import LoggingIntercept
+from pyomo.environ import ConcreteModel, Param, Var, log
+
+# Note that it is important to use the NL writer's NAN
+#nan = nl_writer.nan
```

Comment:
Is this actually meant to be commented out?

Reply:
removed.